src/lib/hash/blake2/blake2b.cpp

```diff
@@ -32,26 +32,58 @@ const uint64_t blake2b_IV[BLAKE2B_IVU64COUNT] = {
 }
 
 BLAKE2b::BLAKE2b(size_t output_bits) :
+   BLAKE2b(nullptr, 0, output_bits)
+   {
+   }
+
+BLAKE2b::BLAKE2b(const SymmetricKey& key, size_t output_bits) :
```

Comment:
This is inconsistent with how the rest of the library treats objects, which are initialized un-keyed, later can be keyed/rekeyed. So I think both this constructor and the second taking keys as bytes should be removed here.

Reply:
I only added them as it's quite common in other libs to support keyed construction for Blake2b, but happy to remove it.